### ISSUE TITLE
Fix Runtime errors in ClientIdPage.js

### ISSUE THREAD

**Issue by eckdanny-osi** [ca-cwds/case-management#160]
Please resolve the js runtime errors thrown at route `/clientid/index` on `master` at f012a01:

```
Warning: validateDOMNesting(...): Text nodes cannot appear as a child of <tbody>.
    in tbody (created by Table)
    in table (created by Table)
    in div (created by Table)
    in Table (created by Ethnicity)
    in div (created by Ethnicity)
    in div (created by Ethnicity)
    in div (created by Cards)
    in div (created by Cards)
    in div (created by Cards)
    in Cards (created by Ethnicity)
    in Ethnicity (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in ClientIdPage
```

```
Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
    in input (created by InputComponent)
    in label (created by InputComponent)
    in div (created by InputComponent)
    in div (created by InputComponent)
    in InputComponent (created by ClientInformation)
    in div (created by ClientInformation)
    in div (created by Cards)
    in div (created by Cards)
    in div (created by Cards)
    in Cards (created by ClientInformation)
    in ClientInformation (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in ClientIdPage
```

```
Warning: A component is changing a controlled input of type string to be uncontrolled. Input elements should not switch from controlled to uncontrolled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
    in input (created by InputComponent)
    in label (created by InputComponent)
    in div (created by InputComponent)
    in div (created by InputComponent)
    in InputComponent (created by ClientInformation)
    in div (created by ClientInformation)
    in div (created by Cards)
    in div (created by Cards)
    in div (created by Cards)
    in Cards (created by ClientInformation)
    in ClientInformation (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in div (created by ClientIdPage)
    in ClientIdPage
```

**Comment by RamuKammagani-osi** [comment]
I can only see "Warning: validateDOMNesting(...): Text nodes cannot appear as a child of <tbody>." but i don't see the rest of two warnings! may i know where you can see.

**Comment by RamuKammagani-osi** [comment]
Fixed all the warning! Can we close this now?

**Comment by eckdanny-osi** [comment]
resolved as of be4e2f1

closing.

**Comment by RamuKammagani-osi** [comment]
Runtime errors shows up again!
